app/assets/stylesheets/application.scss

```diff
@@ -2,3 +2,5 @@
 // for components and views on pages where they are needed.
 
 // https://github.com/alphagov/govuk_publishing_components/blob/main/docs/set-up-individual-component-css-loading.md
+@import "govuk_publishing_components/govuk_frontend_support";
+@import "govuk_publishing_components/component_support";
```

Comment:
Static includes a number of component stylesheets needed by all applications e.g. heading - do some of those need adding here?

Reply:
We've added here a bundled set of components that are used by all layouts in the app. The individual stylesheet for headings will be brought in as an individual component.